src/Reflex/Dom/Xhr.hs

```diff
@@ -50,6 +50,9 @@ import Data.Map (Map)
 import qualified Data.Map as Map
 import Data.Maybe
 import Data.Text (Text)
+import qualified Data.Text as T
+import GHCJS.DOM.XMLHttpRequest (getAllResponseHeaders)
```

Comment:
ghcjs-dom compiled with ghc doesn't export `getAllResponseHeaders`. Probably have to handle this the way the other XMLHttpRequest things are handled in src-ghc/Reflex/Dom/Xhr/Foreign.hs